_pytest/config.py

```diff
@@ -82,6 +82,12 @@ def get_config():
         pluginmanager.import_plugin(spec)
     return config
 
+def get_plugin_manager():
+    """deprecated, backward-compatibility for pytest < 2.8 (see #787)"""
+    warnings.warn('get_plugin_manager() is deprecated, use get_config().pluginmanager instead',
+                  DeprecationWarning, stacklevel=2)
+    return get_config().pluginmanager
```

Comment:
sorry i didn't say this earlier but i think we don't need to bother with deprecating it.  @RonnyPfannschmidt , do you insist to deprecate it?  I think it's ok to offer a method for IDEs and others to get a plugin manager instance.